examples/opentelemetry/docker-compose.yaml

```diff
@@ -69,3 +69,6 @@ services:
     command: ["--config=/etc/otel-collector-config.yaml"]
     ports:
     - "${PORT_UI:-55679}:55679"
+    volumes:
+      - "./otel-collector-config.yaml:/etc/otel-collector-config.yaml"
+      
```

Comment:
all files in envoy are expected to have a newline - it doesnt have to be an empty line - it really is best to configure your editor as suggested by the ci:

```console
  Please fix your editor to ensure:

    - no trailing whitespace
    - no preceding mixed tabs/spaces
    - all files end with a newline
```